resources/sql_scripts/CVManager_CreateTables.sql

```diff
@@ -401,4 +401,25 @@ CREATE TABLE IF NOT EXISTS public.snmp_msgfwd_config
 		ON DELETE NO ACTION
 );
 
+CREATE SEQUENCE public.obu_ota_request_id_seq

```

Comment:
Lets go ahead and make another SQL script that just includes the new additions and name it accordingly. This will make it very straight forward on getting this deployed throughout the CDOT environments.